src/contacts.cs

```diff
@@ -463,16 +1457,30 @@ interface CNContactStore {
 
 		[Export ("unifiedContactsMatchingPredicate:keysToFetch:error:")]
 		[Protected] // we cannot use ICNKeyDescriptor as Apple (and others) can adopt it from categories
-		CNContact[] GetUnifiedContacts (NSPredicate predicate, NSArray keys, out NSError error);
+		[return: NullAllowed]
+		CNContact[] GetUnifiedContacts (NSPredicate predicate, NSArray keys, [NullAllowed] out NSError error);
```

Comment:
Sharpie generate them like this `[NullAllowed] out ...` but, in _C#_ reality, we can't use `null` on an `out` parameter.
Not bad (and not worth changing)

`[return: NullAllowed]` is also not used (by the generator) but it could become useful soon (with C# 8) so it's nice too add them when missing :)

Reply:
Thanks for the info! :)